<launch>
    <param name="/use_sim_time" value="true" /> 
    <node pkg="ndt_imu_localization" name="pointcloud_filter_node" type="pointcloud_filter_node" >
        <!-- pointcloud topic -->
        <param name="point_cloud_topic" value="/velodyne_points" />
        <!-- <param name="point_cloud_topic" value="/kitti/velo/pointcloud" /> -->

        <!-- downsample filter param -->
        <!-- kitti param 0.5 velodyne param 0.2 -->
        <param name="filter_downsample" value="true" />
        <!-- <param name="downsample_resolution" value="0.5" />  -->
        <param name="downsample_resolution" value="0.2" /> 

        <!-- outlier filter param -->
        <param name="filter_outlier" value="false" />
        <param name="outlier_radius" value="1" />
        <param name="outlier_neighbors" value="3" />
    </node>

    <node pkg="tf" type="static_transform_publisher" name="world_map_broadcaster" args="0 0 -0.4 0 0 0 /world /map 10" />
    <node pkg="tf" type="static_transform_publisher" name="lidar_base_broadcaster" args="-0.2 0 -0.4 0 0 0 /lidar_link /base_link 10" />
    <node pkg="tf" type="static_transform_publisher" name="lidar_velodyne_broadcaster" args="0 0 0 0 0 0 /lidar_link /velodyne 10" />

    <node pkg="rviz"  type="rviz"  name="rviz"  args="-d $(find ndt_imu_localization)/rviz/school.rviz"></node>

</launch>